src/models/FGAerodynamics.cpp

```diff
@@ -261,25 +263,25 @@ bool FGAerodynamics::Run(bool Holding)
       vForcesAtCG = vFnativeAtCG;
       break;
     case atStability:   // Convert from stability axes to both body and wind axes
-	   {
-		  FGColumnVector3 vFs;
-		  vFs.InitMatrix();
-		  vFs = vFnative;
-		  vFs(eDrag) *= -1; vFs(eLift) *= -1;
-		  vForces = Ts2b*vFs;
-		  vFw = Ts2w*vFs;
-		  vFs(eDrag) *= -1; vFs(eLift) *= -1;
-		  vFw(eDrag) *= -1; vFw(eLift) *= -1;
-
-		  FGColumnVector3 vFsAtCG;
-		  vFsAtCG.InitMatrix();
-		  vFsAtCG = vFnativeAtCG;
-		  vFsAtCG(eDrag) *= -1; vFsAtCG(eLift) *= -1;
-		  vForcesAtCG = Ts2b*vFsAtCG;
-		  vFwAtCG = Ts2w*vFsAtCG;
-		  vFsAtCG(eDrag) *= -1; vFsAtCG(eLift) *= -1;
-		  vFwAtCG(eDrag) *= -1; vFwAtCG(eLift) *= -1;
-	   }
```

Comment:
There are some other areas in the source code where the indentation is broken. I just mentioned this one as an example.